extensions/crypto/lib.deno_crypto.d.ts

```diff
@@ -95,6 +100,24 @@ interface SubtleCrypto {
     extractable: boolean,
     keyUsages: KeyUsage[],
   ): Promise<CryptoKeyPair | CryptoKey>;
+  importKey(
+    format: "raw",
+    keyData:
+      | Int8Array
```

Comment:
I apologize if this was explained elsewhere: is that a reason you're using this list of types instead of the `BufferSource` type that's already defined in Deno? (Maybe because this approach produces better autocomplete and type error messages?) I am just wondering for the sake of my std/crypto PR, which I will probably update to match what you've done in WebCrypto.

Reply:
Yes this should be updated to `BufferSource`. Typescript just never fixed it https://github.com/Microsoft/TypeScript/issues/23055 :thinking:

Reply:
Interesting! Thanks.